ckanext/oidc_pkce/tests/test_plugin.py

```diff
@@ -1,9 +1,52 @@
+# encoding: utf-8
+
+from unittest.mock import MagicMock
 import pytest
 
 import ckan.plugins as p
 
+from ckanext.oidc_pkce import plugin as plugin_module
+
 
 @pytest.mark.ckan_config("ckan.plugins", "oidc_pkce")
 @pytest.mark.usefixtures("with_plugins")
 def test_plugin():
     assert p.plugin_loaded("oidc_pkce")
+
+
+if p.toolkit.check_ckan_version('2.10'):
```

Comment:
need test user not sso enabled, skip custom logout flow.

Reply:
Unit test is now added, manual testing on test-dev looks good.